samples/hello-world-frontend-plugin.devfile

```diff
@@ -0,0 +1,29 @@
+specVersion: 0.0.1
```

Comment:
According to voting in the following issue https://github.com/eclipse/che/issues/12837
We might approve new devfile file name. I think it makes sense to end samples with `.devfile.yaml`

Reply:
we'll rename them after final decision will be done  (and there will be another > 9999 places to rename)